tests/utils/expressions.js

```diff
@@ -0,0 +1,37 @@
+export const getProductNameExpression = ( productTitle ) =>
+	`a[contains(text(), "${ productTitle }")]`;
+
+export const getQtyItemExpression = ( args ) =>
+	'div[@class="wc-block-cart-item__quantity" and div[@class="wc-block-components-quantity-selector" and ' +
+	getQtyInputExpression( args ) +
+	']]';
+
+export const getQtyInputExpression = ( args = {} ) => {
+	let qtyValue = '';
+
+	if ( args.qty ) {
+		qtyValue = ` and @value="${ args.qty }"`;
+	}
+
+	return (
+		'input[contains(@class, "wc-block-components-quantity-selector__input")' +
```

Comment:
noob question: I've never used x path selectors, what are the benefits over normal selectors?

Reply:
@alexflorisca, I never used it before either. But after doing some research, it appears `XPath` is more powerful than `CSS Selector` when it comes to using complex conditional logic to get a specific element. In my case, what made me ditch `CSS Selector` in this instance is that it can't target an element by the content of its text node.